lua/docs-view.lua

```diff
@@ -44,6 +44,18 @@ local function toggle()
       callback = function()
         if win and vim.api.nvim_win_is_valid(win) then
           local l, c = unpack(vim.api.nvim_win_get_cursor(0))
+          local clients = vim.lsp.buf_get_clients(0)
+
+          local gotHover = false
+          for i = 1, #clients do
+            if clients[i].supports_method("textDocument/hover") then
+              gotHover = true
+              break
+            end
+          end
+          if not gotHover then
+            return
+          end
```

Comment:
could you move this logic one line above? just so `l, c` are defined next to where they are used

Reply:
done.